src/main/java/az/edu/strangers/controller/FamilyController.java

```diff
@@ -84,4 +84,9 @@ public boolean addPet(final Integer index, final Pet pet) {
 
         return familyService.addPet(index, pet);
     }
+
+    @Override
+    public String toString() {
```

Comment:
why we need toString here?

Reply:
for print

Reply:
I think we don't need toString method in this layer